src/components/deck-collection-filters/deck-filters-wrapper.module.css

```diff
@@ -0,0 +1,30 @@
+.search-container {
+  display: flex;
+  width: 100%;
+  padding-bottom: 0.5rem;
+}
+
+.search-input {
+  width: 100%;
+  border-radius: var(--rounded-lg);
```

Comment:
I changed the card search bar radius on main, we probably want to reflect that here as well